include/chia/copy.h

```diff
@@ -17,10 +17,22 @@
 #include <cstdio>
 #include <cstdint>
 
+#ifdef __linux__
+	#include <filesystem>
+	namespace fs = std::filesystem;
+#endif
 
 inline
 uint64_t copy_file(const std::string& src_path, const std::string& dst_path)
 {
+	#ifdef __linux__
+		int plotsize = 109000000000
```

Comment:
are you sure that number fits into an `int` lol
you need `uint64_t` here

Reply:
make it `const uint64_t` too

Reply:
yes I was wondering about that! I guess it would be better to get the real size value but just put a rough value for now

Reply:
it's not a constant anyways, all good